numpy/_core/meson.build

```diff
@@ -1137,6 +1137,12 @@ src_umath = umath_gen_headers + [
 # here. Note that this migration is desirable; we then get the performance
 # benefits for all platforms rather than only for AVX512 on 64-bit Linux, and
 # may be able to avoid the accuracy regressions in SVML.
+
+svml_file_suffix = ['d_la', 's_la', 'd_ha', 's_la']
+if CPU_DISPATCH_NAMES.contains('AVX512_SPR') or CPU_BASELINE_NAMES.contains('AVX512_SPR')
```

Comment:
@seiko2plus does this work? or is there a better way to add svml fp16 sources conditionally?

Reply:
Yes, it should work.
> or is there a better way to add svml fp16 sources conditionally?

Maybe export a new var that combines both lists `CPU_FEATURES_NAMES = CPU_BASELINE_NAMES + CPU_DISPATCH_NAMES`
Then use `in` expression:
```meson
if 'AVX512_SPR' in CPU_FEATURES_NAMES
  error('test me')
endif
```

Reply:
Done.